marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mastering/previewMatchingActivity2/previewMatchingActivity.sjs

```diff
@@ -0,0 +1,18 @@
+const test = require("/test/test-helper.xqy");
+
+const pma = require("/data-hub/5/mastering/preview-matching-activity-lib.xqy");
+
+let uris = []
+for (let uri of cts.uriMatch("/content/5*.json")) {
+  xdmp.log(uri);
+  uris.push(uri);
+}
+
+let options = cts.doc("/steps/matching/patientMatch.step.json").root;
+xdmp.log(options);
+let results = pma.previewMatchingActivity(options, uris, 0);
+xdmp.log(results.actionPreview.length);
+xdmp.log(results);
+[
+  test.assertEqual(19, results.actionPreview.length, "There should be 19 matching pairs")
```

Comment:
I think this should be simplified to expect 2 pairs - like there should be 2 that match, and 1 that does not. And the assertion message should make it clear which 2 are expected and why they should match. 

Otherwise, if this breaks, it'll be difficult to figure out what the problem is and why it broke. This is similar to the challenge we've had with MasterTest - when we don't get 205 merges, it's nearly impossible to figure out why we're not because we don't know exactly which 205 are expected, nor why. 

That should also remove the need for the log statements, which I assume are there to facilitate debugging.